Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -80,6 +82,48 @@ def cmd_fail(*args)
     output
   end
 
+  def setup_test_formula(name, content = nil)
+    formula_path = CoreTap.new.formula_dir/"#{name}.rb"
+    @formula_files << formula_path
+
+    case name
+    when "testball"
+      content = <<-EOS.undent
+        desc "Some test"
+        homepage "https://example.com/#{name}"
+        url "file://#{File.expand_path("..", __FILE__)}/tarballs/#{name}-0.1.tbz"
+        sha256 "#{TESTBALL_SHA256}"
+
+        option "with-foo", "Build with foo"
+        #{content}
```

Comment:
I added this line solely for `test_options`, but only now realized that it seems wrong to nest `#{content}` inside a declaration for `content`. Still, the test seems to pass...